doc/lsst.afw.image/exposure-fits-metadata.rst

```diff
@@ -129,116 +142,119 @@ HDU 0
      - float
      - deg
      - `VisitInfo.getObservatory`
-     - Longitude of telescope.
+       Longitude of telescope.
 
    * - ``OBS-LAT``
      - float
      - deg
      - `VisitInfo.getObservatory`
-     - Latitude of telescope (positive eastward).
+
+       Latitude of telescope (positive eastward).
 
    * - ``OBS-ELEV``
      - float
      - m
      - `VisitInfo.getObservatory`
-     - Geodetic elevation of telescope (meters above reference spheroid).
+
+       Geodetic elevation of telescope (meters above reference spheroid).
 
    * - ``AIRTEMP``
      - float
      - C
      - `VisitInfo.getWeather`
-     - Air temperature.
+
+       Air temperature.
 
    * - ``AIRPRESS``
      - float
      - Pascals
      - `VisitInfo.getWeather`
-     - Air pressure.
+
+       Air pressure.
 
    * - ``HUMIDITY``
      - float
      - %
      - `VisitInfo.getWeather`
-     - Relative humidity.
+
+       Relative humidity.
 
    * - ``FLUXMAG0``
      - float
      - ADU
      - `Calib.getFluxMag0`
-     - Flux of a zero-magnitude object.
+
+       Flux of a zero-magnitude object.
 
    * - ``FLUXMAG0ERR``
      - float
      - ADU
      - `Calib.getFluxMag0Err`
-     - Error in the flux of a zero-magnitude object.
+
+       Error in the flux of a zero-magnitude object.
 
    * - ``FILTER``
      - str
      -
      - `Filter.getName`
-     - Name of filter.
+
+       Name of filter.
 
    * - ``AR_HDU``
      - int
      -
-     -
      - HDU containing the archive used to store ancillary objects
 
    * - ``COADD_INPUTS_ID``
      - int
      -
-     -
      - Archive ID for coadd inputs catalogs
 
    * - ``AP_CORR_MAP_ID``
      - int
      -
-     -
      - Archive ID for aperture correction map
 
    * - ``PSF_ID``
      - int
      -
-     -
      - Archive ID for the Exposure's main Psf
 
    * - ``WCS_ID``
      - int
      -
-     -
      - Archive ID for the Exposure's main Wcs
 
    * - ``VALID_POLYGON_ID``
      - int
      -
-     -
      - Archive ID for the Exposure's valid polygon
 
 HDUs 1 to 3
 ===========
 
 .. list-table::
-   :widths: 2 1 1 3 5
+   :widths: 2 1 1 5
    :header-rows: 1
 
    * - Keyword
      - Format
      - Units
-     - LSST object and accessor [1]_
-     - Description
+     - Accessor [1]_ and description
 
    * - ``LTV1``
```

Comment:
LTV1 and LTV2 no longer hold XY0 (those keywords are still written, though we probably should not mention that, as LSST code never reads them). XY0 is now encoded as WCS "A" using the following keywords:
- CRPIX[12]A = 1
- CRVAL[12]A = xy0
- CTYPE[12]A = "LINEAR"
- CUNIT[12]A = "PIXEL"
where [12] means 1 or 2, so CRPIX1 = CRPIX2 = 1, CRVAL1A = xy0[0], CRVAL2A = xy0[1], etc.

If you are willing, please update the documentation accordingly.

Reply:
<img width="666" alt="screen shot 2018-07-13 at 11 02 37 am" src="https://user-images.githubusercontent.com/349384/42701671-d3641a4c-868c-11e8-9dad-a122d897ed9e.png">

@r-owen Here's my update. Does this presentation make sense?

Reply:
(I'll add CUNIT2A)

Reply:
That looks great. Thank you very much for updating that information!